plugin/etcd/plugin.go

```diff
@@ -46,10 +45,10 @@ func main() {
 		Fields: []plugin.Field{
 			plugin.Field{
 				Mode:     "target",
-				Name:     "endpoints",
+				Name:     "etcd_url",
 				Type:     "string",
 				Title:    "Endpoint",
-				Help:     "IP addresses of the etcd nodes in the cluster",
+				Help:     "It is the upstream etcd client endpoint",
```

Comment:
Change this to "The client URL(s) of the etcd cluster.  Separate multiple URLs with commas."